src/examples/mailchimp/src/backend/index.ts

```diff
@@ -15,10 +15,8 @@ if (!MAILCHIMP_LIST_ID) throw new Error('MAILCHIMP_LIST_ID is required in enviro
 if (!MAILCHIMP_PROJECTS_CATEGORY_ID)
 	throw new Error('MAILCHIMP_PROJECTS_CATEGORY_ID is required in environment');
 
-const resolvers = [interest, category, mailingList, member];
-
-const graphweaver = new Graphweaver({
+export const graphweaver = new Graphweaver({
 	resolvers,
 });
 
-exports.handler = graphweaver.handler();
+export const handler = graphweaver.handler();
```

Comment:
We're exporting the graphweaver, handlers, and resolvers now. Are we importing them anywhere?

Reply:
No need to export the resolvers and yes we need to export both `graphweaver` and the `handler`.

The `handler` is imported by the lambda that runs the code.

The `graphweaver` instance is imported by the types generation to get the GraphQL schema.